src/main/java/org/wiremock/webhooks/Webhooks.java

```diff
@@ -128,7 +128,7 @@ private WebhookDefinition applyTemplating(
         webhookDefinition.getExtraParameters() != null
             ? webhookDefinition.getExtraParameters()
             : Collections.<String, Object>emptyMap());
-    model.put("originalRequest", RequestTemplateModel.from(serveEvent.getRequest()));
+    model.put("originalRequest", RequestTemplateModel.from(serveEvent));
```

Comment:
Could we inject the template engine and call that to build the model instead?

That way we also get the extra model elements provided by extensions.

Reply:
Sounds good.  Will take a look

Reply:
@tomakehurst this would add the `request` key to the model along side the `originalRequest`.  I assume this wasn't added originally for a reason.  Would we want to remove this from the model that the webhooks use?